src/Runner/TaskRunnerContext.php

```diff
@@ -22,16 +22,26 @@ class TaskRunnerContext
      */
     private $testSuite = null;
 
+    /**
+     * @var string[]
+     */
+    private $tasks = null;
+
     /**
      * TaskRunnerContext constructor.
      *
-     * @param ContextInterface   $taskContext
+     * @param ContextInterface $taskContext
      * @param TestSuiteInterface $testSuite
+     * @param string[]|null $tasks
      */
-    public function __construct(ContextInterface $taskContext, TestSuiteInterface $testSuite = null)
+    public function __construct(ContextInterface $taskContext, TestSuiteInterface $testSuite = null, $tasks = null)
```

Comment:
Can you make $tasks a required array?
I would prefer to always pass in a list of tasks.
(This is a BC break, but I don't think it will cause too much damage since it is an internal DTO.)

Reply:
Like this?

```php
    /**
     * TaskRunnerContext constructor.
     *
     * @param ContextInterface $taskContext
     * @param string[] $tasks
     * @param TestSuiteInterface $testSuite
     */
    public function __construct(ContextInterface $taskContext, array $tasks, TestSuiteInterface $testSuite = null)
```

Reply:
yes